modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/jmx/PortalManager.java

```diff
@@ -40,7 +40,12 @@ public PortalManager() throws NotCompliantMBeanException {
 
 	@Override
 	public long getAverageTime() {
-		return _serverSummaryStatistics.getAverageTime();
+		try {
```

Comment:
Hey @Valmir-unicap, please declare a `throws` statement instead of surrounding this with a `try-catch` block.